app/components/markdown/index.js

```diff
@@ -290,6 +292,14 @@ export default class Markdown extends PureComponent {
         return rendered;
     }
 
+    transformLinkUri = (uri) => {
+        const filteredUri = uriTransformer(uri);
+
+        const parsed = urlParse(filteredUri, {});
```

Comment:
I think this is done automatically by commonmark-react-renderer, isn't it?

https://github.com/hmhealey/commonmark-react-renderer/blob/master/src/commonmark-react-renderer.js#L411